cmd/rook/ceph/osd.go

```diff
@@ -326,9 +328,13 @@ func removeOSDs(cmd *cobra.Command, args []string) error {
 		return errors.Wrapf(err, "failed to parse --preserve-pvc flag")
 	}
 
-	exitIfNotSafe := false
+	promptIfNotSafeBool, err := strconv.ParseBool(promptIfNotSafe)
+	if err != nil {
+		return errors.Wrapf(err, "failed to parse --prompt-if-not-safe flag")
+	}
+
 	forceRemovalCallback := func(x int) (bool, bool) {
-		return forceOSDRemovalBool, exitIfNotSafe
```

Comment:
In this callback the prompt needs to be implemented. 
```
forceRemovalCallback := func(x int) (bool, bool) {
		if promptIfNotSafe {
			// show the prompt
			// return the value needed depending on the response 
		} 
		return forceOSDRemovalBool, exitIfNotSafe
```

Reply:
How I can read user input in the job logs when I'm testing with job it is not asking for user input and canceling it instead
```
2023-12-06 05:42:37.906232 I | cephcmd: Are you sure you want to remove osd ID 0?
2023-12-06 05:42:37.906343 E | cephcmd: cancelled
```

Reply:
In the job we don't expect the prompt to ever show, right? The user doesn't have the ability to interact from the job, so if they want the prompt they need to use a CLI tool. The job should never use the prompt flag and just look at the force flag.

Reply:
I think getting ack will not be possible as we run the rook command to purge-osd inside the rook operator only.